plugins/group-focus-hover/index.js

```diff
@@ -0,0 +1,10 @@
+module.exports = () => ({ addVariant, e }) => {
+  addVariant('group-focus-hover', ({ modifySelectors, separator }) => {
+    modifySelectors(({ className }) => {
+      return `
+        .group:focus .${e(`group-focus-hover${separator}${className}`)},
+        .group:hover .${e(`group-focus-hover${separator}${className}`)}
+      `
+    })
+  })
```

Comment:
Thoughts on following the pattern used by Tailwind core for `group-hover`? That's what I used for parent-open and parent-expanded. https://github.com/tailwindcss/tailwindcss/blob/6343d1ec7c1d6eb21cce8b2ded02bc5f700e99f6/src/lib/substituteVariantsAtRules.js#L26-L38

Reply:
I would love to know if there's a difference if `modifySelectors()` works as-is.
The docs seem to encourage the more complex approach only if you can't achieve it with `modifySelectors()`?
https://tailwindcss.com/docs/plugins#basic-variants

Reply:
I have no thoughts on one versus the other here. Let me know if I should change it to match our preference? 🤷

Reply:
🤷‍♂️ Not sure what the difference is. I just copied/pasted/modified the TW source for `group-hover` when I made my parent-expanded and parent-open plugins. I'm fine with either approach.